tasks/deploy.yml

```diff
@@ -45,14 +45,14 @@
 
 - name: Update the Preferences Server base config file's authDBServerPort
   replace:
-    dest: "{{ nodejs_app_install_dir }}/{{ preferences_server_config_dir }}/gpii.preferencesServer.config.base.json"
+    dest: "{{ nodejs_app_install_dir }}/{{ preferences_server_config_dir }}/{{ preferences_server_environment }}.json"
```

Comment:
Looks like these should be single quotes for consistency with the rest of the file. :)